### PR TITLE
Initial testing & automatic translation

### PR DESCRIPTION
**Pull request by Nolski**
First pass at doing automatic testing along with corresponding unit tests.

### CODE REVIEW

**Comment by Nolski** [comment]
This is blocked on #11

File: js/package.json [hunk 1]
```diff
@@ -14,16 +14,20 @@
   },
   "dependencies": {},
   "devDependencies": {
+    "@babel/core": "^7.7.7",
```
Comment (Nolski) [inline js/package.json:17]: This is all needed for our tests.

File: js/package.json [hunk 2]
```diff
@@ -14,16 +14,20 @@
...
+    "@babel/preset-env": "^7.7.7",
     "@commitlint/cli": "latest",
     "@commitlint/config-conventional": "latest",
     "@joseph184/rollup-plugin-node-builtins": "^2.1.4",
-    "ava": "latest",
+    "@tensorflow/tfjs": "^1.2.5",
```
Comment (Nolski) [inline js/package.json:22]: added to devDependencies as well as peer deps aren't included in tests.

File: js/package.json (hunk 1 above)
Comment (Nolski) [inline js/package.json:17]: This is all needed for our tests.

**Review by cereallarceny — APPROVED** [review]

**Pull request merged**